pyproject.toml

```diff
@@ -84,8 +88,17 @@ dependencies = [
   # and bug fixes. We fix the version for improved stability and manually update
   # if necessary.
   "chipwhisperer@https://github.com/newaetech/chipwhisperer-minimal/archive/2643131b71e528791446ee1bab7359120288f4ab.zip",
+
+  # Test
+  "hello @ file://${PROJECT_ROOT}/util/hello",
+
+  # OpenTitan python package
+  "opentitanlib @ file://${PROJECT_ROOT}/util/python",
 ]
 
 [tool.setuptools]
 # This is actually not a python project, we just use pyproject.toml to manage dependencies.
 py-modules = []
+
+[tool.uv.workspace]
```

Comment:
Interesting concept! I've not seen that UV feature before... perhaps worth looking into.